examples/direct_fidelity_estimation.py

```diff
@@ -207,7 +208,7 @@ def direct_fidelity_estimation(circuit: cirq.Circuit, qubits: List[cirq.Qid],
     Args:
         circuit: The circuit to run the simulation on.
         qubits: The list of qubits.
-        noise: The noise model when doing a simulation.
+        sampler: Either a noisy simulator or an engine.
```

Comment:
I just noticed: you're still referring to the old 'noise' parameter on line 215 below, could you update the comment appropriately?

Reply:
Done